SPECS/kernel/CVE-2024-24859.nopatch

```diff
@@ -0,0 +1,3 @@
+CVE-2024-24859 - in version 6.6.26.1 (note that this CVE is not mentioned in the description but is fixed by this patch)
+upstream: 7835fcfd132eb88b87e8eb901f88436f63ab60f7
```

Comment:
I see this as the fixing patch for this one? https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v6.6.35&id=f56e715ef1c19c42c6aa6cb9280947dea13aab2e

Reply:
nevermind, got this mixed up with CVE-2024-24860